fdbserver/fdbserver.actor.cpp

```diff
@@ -1111,6 +1112,23 @@ struct CLIOptions {
 private:
 	CLIOptions() = default;
 
+	void parseEnvInternal() {
+		for (std::string knob : getEnvironmentKnobOptions()) {
+			auto pos = knob.find_first_of("=");
+			if (pos == std::string::npos) {
+				fprintf(stderr,
+				        "Error: malformed environment knob option: %s%s\n",
+				        ENVIRONMENT_KNOB_OPTION_PREFIX,
+				        knob.c_str());
```

Comment:
I suppose this should also include (instead of or in addition to the print statement) a `SevWarnAlways` trace event that is similar to the existing `InvalidKnobValue` event. That way we can also obtain this info from the trace logs if needed.

Reply:
I will leave the print statement, as this appears in the `joshua tail` output